apps/sms/views/inbox/js/inbox.js

```diff
@@ -697,77 +704,96 @@ var InboxView = {
     });
   },
 
-  createThread: function inbox_createThread(record) {
-    // Create DOM element
-    var li = document.createElement('li');
-    var timestamp = +record.timestamp;
-    var type = record.lastMessageType;
-    var participants = record.participants;
-    var number = participants[0];
+  /**
+   * This method is responsible for adding or updating the data that could
+   * change in a node that's not moving in the same time.
+   *
+   * @param {Node} node The Node to update.
+   * @param {Object} record An object that holds the conversation informations.
+   */
+  updateConversationContent(node, record) {
     var id = record.id;
-    var bodyHTML = record.body;
-    var thread = Threads.get(id);
-    var iconLabel = '';
 
-    // A new conversation "is" a draft
+    var thread = Threads.get(id);
     var isDraft = thread.isDraft;
-
-    // A an existing conversation has draft.
     var draft = thread.getDraft();
 
-    if (!isDraft && draft) {
-      // If the draft is newer than the message, update
-      // the body with the draft content's first string.
-      if (draft.timestamp >= record.timestamp) {
-        bodyHTML = draft.content.find(function(content) {
-          if (typeof content === 'string') {
-            return true;
-          }
-        });
-        type = draft.type;
-      }
-    }
+    var body = record.body;
+    var type = record.lastMessageType;
 
-    bodyHTML = Template.escape(bodyHTML || '');
```

Comment:
now we use textContent to set the body, so we don't need to escape this anymore.